### ISSUE TITLE
Qualitative region description regions need same tolerances as the shape names

### ISSUE THREAD

**Issue by terracoda** [phetsims/quadrilateral#181]
Honestly, I do not know how to deal with "tolerances" for the visual design over in the description design.

We are currently adjusting things, so not sure if this is short term issue or a long term issue. In the following screen shot, the description do not describe Corner A and Corner see as "equal", but rather "comparable".
<img width="578" alt="image" src="https://user-images.githubusercontent.com/1902116/181328267-0d3e0d2e-ee67-4938-ac9c-a07e4cae0097.png">

It seems to me, that if there is tolerance logic to visually define a parallelogram (or any other shape), we would need to apply those same tolerances to the description of the properties that describe any part of that shape. So in this case, Corners A and C should be described as equal.

Any thoughts on this? For example, is it a short or long term issue?

**Comment by terracoda** [comment]
And this can get super complicated if tolerances are different based on input. 
Keyboard input is very precise.

**Comment by terracoda** [comment]
Similar issue here
<img width="580" alt="image" src="https://user-images.githubusercontent.com/1902116/181347425-b66ec318-0ad8-4be2-909e-62968ac0f6f8.png">

The descriptions of each side and corner are not described as equal to its corresponding opposite. This makes it difficult for any deduction on what properties define a shape.

**Comment by BLFiedler** [comment]
Tolerance "checkers" for the shape detection/visuals are not implemented for description yet, but will be.

Keeping the issue up as a to do item for implementation.

**Comment by jessegreenberg** [comment]
Now that #186 is fixed I am having a harder time creating these cases. But I found one by dragging a side so that the tolerance values are infinite:
![image](https://user-images.githubusercontent.com/6396244/183689824-bf43dbe0-6a6b-47ae-a380-e98885b0aa35.png)
![image](https://user-images.githubusercontent.com/6396244/183689885-2eae51b7-5642-4717-9f56-a3432208d2f7.png)

**Comment by jessegreenberg** [comment]
I am not actually sure what to check for here with tolerances. It could be as simple as saying "If the shape is a parallelogram, make sure the description uses "equal angle" descriptions". But I don't know if there are other shapes that have incorrect descriptions in this case. It would be helpful to check in on this during meetings.

**Comment by jessegreenberg** [comment]
This might be improved by #188, lets return to this once that is done.

**Comment by jessegreenberg** [comment]
We can come back to this now that #188 is settled. Console commands to reproduce the case in https://github.com/phetsims/quadrilateral/issues/181#issuecomment-1209524741
```
window.simModel.quadrilateralShapeModel.vertexA.positionProperty.value = new phet.dot.Vector2( -0.25, 0.2375 );
window.simModel.quadrilateralShapeModel.vertexB.positionProperty.value = new phet.dot.Vector2( 0.9375, -0.1 );
window.simModel.quadrilateralShapeModel.vertexC.positionProperty.value = new phet.dot.Vector2( 1.1875, -0.85 )
window.simModel.quadrilateralShapeModel.vertexD.positionProperty.value = new phet.dot.Vector2( 0, -0.5 )
```

**Comment by jessegreenberg** [comment]
I think I still need help on this from https://github.com/phetsims/quadrilateral/issues/181#issuecomment-1209570985.

**Comment by jessegreenberg** [comment]
Master doesn't let this happen currently, here is a new set of vertices that have this problem:
```
window.simModel.quadrilateralShapeModel.vertexA.positionProperty.value
Vector2 {x: -1.125, y: 0.8625}
window.simModel.quadrilateralShapeModel.vertexB.positionProperty.value
Vector2 {x: 1.25, y: 0}
window.simModel.quadrilateralShapeModel.vertexC.positionProperty.value
Vector2 {x: 1.25, y: -0.5}
window.simModel.quadrilateralShapeModel.vertexD.positionProperty.value
Vector2 {x: -1.125, y: 0.35}
```

And to set from console:
```
window.simModel.quadrilateralShapeModel.vertexA.positionProperty.value = new phet.dot.Vector2( -1.125, 0.8625 );
window.simModel.quadrilateralShapeModel.vertexB.positionProperty.value = new phet.dot.Vector2( 1.25, 0 );
window.simModel.quadrilateralShapeModel.vertexC.positionProperty.value = new phet.dot.Vector2( 1.25, -0.5 )
window.simModel.quadrilateralShapeModel.vertexD.positionProperty.value = new phet.dot.Vector2( -1.125, 0.35 )
```

**Comment by BLFiedler** [comment]
Let's discuss this today in more detail to determine the mismatch in description and shape detection.

**Comment by jessegreenberg** [comment]
This is a problem for the same reason that the "right angle indicators' show up incorrectly sometimes. A fix for that issue may be applied here.

Also, Voicing is going to change significantly so I am not sure this will still be relevant with new designs. Lets touch base next meeting.

**Comment by terracoda** [comment]
With the latest changes to the model. We should not be having too many issues.

It might be time to close this issue soon, though we still have an issue with right angles and I just opened #212 so there are still things we are watching out for.

**Comment by BLFiedler** [comment]
I believe this issue is being solved for specific cases elsewhere (e.g., #262 ). Unless there is an underlying bigger picture issue still? 

Passing to @terracoda to see if we should close this issue.

**Comment by terracoda** [comment]
Yes, this issue can be closed. Hopefully, all tolerance related issues will be resolved in the newer issues as lot of changes have happened since this issue was posted.
